woocommerce/woocommerce-android
# [Product AI v2] Generate Again button + Feedback banner logic

Closes: #12007  Closes: #12025

⚠️ Please don't merge until the base branch is `trunk`.

### Description
This PR adds two changes:
1. It adds the "Generate Again" button.
2. It adds logic to handle the "Feedback banner" buttons.

### Testing information
1. Make sure you are signed in using a Jetpack website.
2. Open the AI product creation form.
3. Enter some keywords, then proceed.
4. On the preview screen, when the generation finishes, click on "Generate Again"
5. Confirm this generates a new version of the product.
6. Click on one of the feedback banner buttons.
7. Confirm the following is logged to logcat: `Tracked: product_ai_feedback, Properties: {"source":"product_creation","is_useful":...}`
8. Confirm that the feedback banner was dismissed.

### Images/gif
<img src=https://github.com/user-attachments/assets/64ec71fa-ec12-45c2-bae7-4a5e4b2be1d8 width=400 />
